### PR TITLE
feat(take-a-break): Custom snooze time for take-a-break notification

### PR DESCRIPTION
**Pull request by kwongtn**
# Description

Default snooze value during being prompted for taking breaks after set N time is 15 mins.
This enhancement (hopefully) adds the function to allow users to define the snooze time.

## Issues Resolved

None that I know of

## Check List
Saw that testing was disabled, so did not write (I have yet to learn anyways 😝)
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: src/assets/i18n/zh.json [hunk 1]
```diff
@@ -1052,6 +1052,7 @@
       "IS_FOCUS_WINDOW": "有提醒时激活本应用窗口（仅限桌面）",
       "IS_LOCK_SCREEN": "休息时间到时锁定屏幕（仅限桌面）",
       "MESSAGE": "休息一下",
+      "SNOOZE_TIME": "[TRANSLATION REQUIRED] Snooze time when prompted to take a break",
```
Comment (johannesjo) [inline src/assets/i18n/zh.json:1055]: It's best if you let those empty so my auto translate script can pick up on those.

**Comment by johannesjo** [comment]
Hi there! That's a nice addition! Thank you very much! Just one thing: Could you remove those "[TRANSLATION REQUIRED] " strings? They would confuse my auto translate script :)

**Comment by kwongtn** [comment]
Hi!
I've edited the commit accordingly 😁

File: src/app/features/time-tracking/take-a-break/take-a-break.service.ts [hunk 2]
```diff
@@ -211,21 +211,24 @@ export class TakeABreakService {
...
+          time: msToString(cfg.takeABreak.takeABreakSnoozeTime),
         },
         action: {
           label: T.F.TIME_TRACKING.B.ALREADY_DID,
           fn: () => this.resetTimerAndCountAsBreak(),
         },
         action2: {
           label: T.F.TIME_TRACKING.B.SNOOZE,
-          fn: () => this.snooze(),
+          fn: () => this.snooze(cfg.takeABreak.takeABreakSnoozeTime * 60 * 1000),
```
Comment (johannesjo) [inline src/app/features/time-tracking/take-a-break/take-a-break.service.ts:222]: The `* 60 * 1000` won't work as intended. The duration field should be already in milliseconds.

File: src/assets/i18n/es.json [hunk 3]
```diff
@@ -1052,6 +1052,7 @@
       "IS_FOCUS_WINDOW": "Enfocar la ventana de la aplicación cuando el recordatorio está activo (solo para escritorio)",
       "IS_LOCK_SCREEN": "Bloquear la pantalla cuando se debe tomar un descanso (solo para escritorio)",
       "MESSAGE": "Tomar un mensaje de descanso",
+      "SNOOZE_TIME": "",
```
Comment (johannesjo) [inline src/assets/i18n/es.json:1055]:
```suggestion
```
This needs to be gone completley :)

File: src/assets/i18n/fa.json [hunk 4]
```diff
@@ -1052,6 +1052,7 @@
       "IS_FOCUS_WINDOW": "Focus app window when reminder is active (desktop only)",
       "IS_LOCK_SCREEN": "Lock screen when a break is due (desktop only)",
       "MESSAGE": "Take a break message",
+      "SNOOZE_TIME": "",
```
Comment (johannesjo) [inline src/assets/i18n/fa.json:1055]:
```suggestion
```

**Review by johannesjo — CHANGES_REQUESTED** [review]

**Comment by kwongtn** [comment]
I've edited the commits 😁 
Thank you very much for your patience with this 😥

**Comment by johannesjo** [comment]
No Worries! Thank you very much for your work! :)

**Pull request merged**